hil/cli/port.py

```diff
@@ -11,9 +13,34 @@ def port():
 @port.command(name='show')
 @click.argument('switch')
 @click.argument('port')
-def port_show(switch, port):
+@click.option('--json', 'jsonout', is_flag=True)
+def port_show(switch, port, jsonout):
     """Show what's connected to <port>"""
-    print client.port.show(switch, port)
+
+    raw_output = client.port.show(switch, port)
+
+    if jsonout:
+        json_output = json.dumps(raw_output)
+        print(json_output)
+        return
```

Comment:
I could probably factor the 3 lines. If I refactor the check `if jsonout` into a helper function then I might have to come back to the caller in case the check fails.